content/ros/concepts.md

```diff
@@ -4,5 +4,7 @@ Responsible: Kiran Vasudev | Pranav
 
 Content:
 * Why ROS?
-* Introduction to topics, nodes, messages...
+* Introduction to topics, nodes, messages,rqt graph...
+* Basic commands related to the concepts discussed
```

Comment:
The commands will be covered in a different session in depth. 
The ROS Concepts session should focus mostly on theory, see:
* http://www.clearpathrobotics.com/assets/guides/ros/Intro%20to%20the%20Robot%20Operating%20System.html
* https://www.allaboutcircuits.com/technical-articles/an-introduction-to-robot-operating-system-ros/ (Only the concepts, not the practical part)
* https://www.ethz.ch/content/dam/ethz/special-interest/mavt/robotics-n-intelligent-systems/rsl-dam/ROS2017/lecture1.pdf

Reply:
Here are a few other resources:
* https://courses.cs.washington.edu/courses/cse466/11au/calendar/ros_cc_1_intro-jrsedit.pdf
* https://www.cse.sc.edu/~jokane/agitr/agitr-letter.pdf 

Don't forget to cite!

Reply:
I agree, please change the content and add your slide set for the next review